tests/dogtag/pytest-ansible/pytest/kra/bugzilla/test_bug_1392616_kra_key_recovery_cli_generates_p12_file.py

```diff
@@ -35,6 +35,7 @@
 userop = utils.UserOperations(nssdb=constants.NSSDB, db_pass=constants.CLIENT_DATABASE_PASSWORD)
 template_name = "retrieveKey.template"
 cert_file = "b64_cert"
+transport_file = '/tmp/transport.pem'
```

Comment:
@dpuniaredhat Just remove this 'transport_file = '/tmp/transport.pem'' as discussed.

Reply:
Fixed, thank you :+1: